### ISSUE TITLE
`make all` errors

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-workstation#287]
## Description

When running `make all` in `Dom0` I see two errors: 
* `The following requisites were not found: require: pkg: qubes-template-whonix-gw-14` 
* `The following requisites were not found: require: pkg: qubes-template-whonix-ws-14`

#### STR
1. from `sd-dev` pull the latest changes from `master` branch
2. from `Dom0`, set the env vars needed for `make clone`
3. `make clone`
4. cd into cloned repo
5. update config.json with the correct server info
6. `make all`

#### Expected
No errors

#### Actual output
```
...
ID: sd-whonix
Function: qvm.vm
Result: False
Comment: The following requisites were not found: require: pkg: qubes-template-whonix-gw-14
...
ID: sd-proxy
Function: qvm.vm
Result: False
The following requisites were not found: require: pkg: qubes-template-whonix-ws-14
...
Succeeded: 54 (changed=12)
Failed:          2
...
```

**Comment by creviera** [comment]
In Dom0, when I run `dnf list | grep whonix-`, I see:
```
qubes-template-whonix-gw-14.noarch    4.0.1-201811291216   @anaconda 
qubes-template-whonix-gw-15.noarch    4.0.1-201906232114   @qubes-dom0-cached 
qubes-template-whonix-gw-14.noarch    4.0.1-201811291216   @anaconda 
qubes-template-whonix-gw-15.noarch    4.0.1-201906232114   @qubes-dom0-cached 
```

**Comment by creviera** [comment]
@emkll you had me run a qvm-remove command or something, I can't remember what it was atm, but that might be what caused the errors? can you try running that removal command on the sd-whonix vm like we did yesterday on my machine?

**Comment by emkll** [comment]
Thanks for the detailed report @creviera . I've poked around trying to call qubes-dom0-update to download qubes-template-whonix-ws-14 and can no longer download that package in dom0, something must have changed, and the package can no longer be found.
Since qubes-template-whonix-14-{ws,gw} packages are shipped with the Qubes 4.0.1 isos, I think that logic can safely be removed, I will try to push a fix for you to test.
